packages/core/src/augury-core.ts

```diff
@@ -1,29 +1,50 @@
-import { AuguryEventAssembler, ChannelManager } from './channels';
+import { Subscription } from './events/event-emitters';
 import { HistoryManager } from './history';
 import { PluginManager } from './plugins';
 import { Plugin } from './plugins';
 import { Probe, ProbeManager } from './probes';
 import { AuguryEventProjection } from './projections';
 
 export class AuguryCore {
-  public readonly historyManager: HistoryManager;
   private readonly probeManager: ProbeManager;
-  private readonly channelManager: ChannelManager;
+  private readonly historyManager: HistoryManager;
   private readonly pluginManager: PluginManager;
 
   constructor(probes: Probe[], plugins: Plugin[], ngZone, ngModule) {
     this.probeManager = new ProbeManager(probes, ngZone, ngModule);
-    this.channelManager = new ChannelManager();
     this.historyManager = new HistoryManager();
     this.pluginManager = new PluginManager(plugins, this);
     this.probeManager.subscribe(event => this.historyManager.addEvent(event));
   }
 
-  public createSimpleChannel(projection: AuguryEventProjection<any>) {
-    return this.channelManager.createSimpleChannel(this.probeManager, projection);
+  public projectRealTimeEvents<Output>(
```

Comment:
I'm wondering if this name `registerEventProjection` might imply the intention of this a bit better. The non real-time could be `projectResultsFromHistory`?